code/modules/research/designs/equipment_designs.dm

```diff
@@ -242,6 +242,14 @@
 	materials = list(MAT_METAL = 4500, MAT_SILVER = 2500, MAT_TITANIUM = 3500)
 	category = list("Equipment")
 
+/datum/design/minicrowbar
+	name = "Miniature Titanium Crowbar"
+	desc = "A lightweight and portable version of the Crowbar that fits into smaller storages."
+	id = "minicrowbar"
+	req_tech = list("materials" = 3, "engineering" = 4)
```

Comment:
I forget, does this mean this can be destroyed for these tech levels?

Reply:
Line 57 in code/objetcs/items/tools/crowbar.dm says it has an origin tech  of materials 2, which does allow it to be destroyed. This is just tech levels required to craft them.